### PR TITLE
WIP -- sample data for the new 'modal' display strategy within the Ap…

### PR DESCRIPTION
**Pull request by drewwills**
…ereo Notification portlet;  need a release of that module and some general cleanup of this data

##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

Providing this work as a PR because I expect to collaborate with @ChristianMurphy, perhaps others.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by drewwills** [comment]
Is there someone who could whip up a generic/example Terms of Service notification, add it , and configure it within this PR?

File: data/quickstart/portlet-definition/notification-modal.portlet-definition.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <timeout>20000</timeout>
+    <portlet-descriptor>
+        <ns2:webAppName>/NotificationPortlet</ns2:webAppName>
+        <ns2:portletName>Notifications</ns2:portletName>
+    </portlet-descriptor>
+    <lifecycle>
+        <entry name="PUBLISHED" user="admin">2017-09-06T15:57:28.527-07:00</entry>
+    </lifecycle>
+    <category>Information</category>
+    <group>Everyone</group>
```
Comment (bjagg) [inline data/quickstart/portlet-definition/notification-modal.portlet-definition.xml]: Should this be for authenticated users only?

**Comment by ChristianMurphy** [comment]
CI failure is related to https://www.traviscistatus.com/incidents/4qylrqvy50gy

> Repositories running on travis-ci.org and travis-ci.com are experiencing an increase in errored builds. We are investigating and will update as soon as we can.
> Sep 07, 2017 - 09:36 UTC

File: data/quickstart/portlet-definition/notification-modal.portlet-definition.xml (hunk 1 above)
Comment (drewwills) [reply data/quickstart/portlet-definition/notification-modal.portlet-definition.xml]: Um... maybe?  (I recognize this setting doesn't match the fragment it has been placed on.)

Will adopters use modal notifications for general, non-targeted emergencies?

**Comment by ChristianMurphy** [comment]
:construction_worker_man: :no_entry: :checkered_flag: AppVeyor CI failure is unrelated to this change, but is something we want to keep an eye on.

```
:tomcatClearLogs
Putting task artifact state for task ':tomcatClearLogs' into context took 0.0 secs.
Executing task ':tomcatClearLogs' (up-to-date check took 0.0 secs) due to:
  Task has not declared any outputs.
Deleting catalina.2017-09-07.log
:tomcatClearLogs FAILED
:tomcatClearLogs (Thread[Daemon worker,5,main]) completed. Took 0.371 secs.
FAILURE: Build failed with an exception.
* Where:
Script 'C:\projects\uportal-start\gradle\tasks\tomcat.gradle' line: 143
* What went wrong:
Execution failed for task ':tomcatClearLogs'.
> Unable to delete file: C:\projects\uportal-start\.gradle\tomcat\logs\catalina.2017-09-07.log
```

**Comment by drewwills** [comment]
The version of Notification that this PR depends on is in.

This PR only needs some properly-configured sample data -- a notice that looks appropriate as a modal dialog.

**Comment by ChristianMurphy** [comment]
Hey @drewwills :wave: 
It looks like this PR hasn't gained any traction and has a failing build. :no_entry: 
I'm going to close this PR, feel free to re-open or open a new PR when there is renewed interest. :+1: 
Thanks for pulling the PR together :bowing_man:
Hope to see a mergeable version soon. :smile:

**Pull request closed without merging**